wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WContainer.java

```diff
@@ -12,17 +12,12 @@
  */
 public class WContainer extends WBeanComponent implements MutableContainer {
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public void add(final WComponent component) {
 		super.add(component);
 	}
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.